### PR TITLE
(RE-5681) Add the ability to ship nuget packages

### PR DESCRIPTION
**Pull request by melissa**

### CODE REVIEW

File: tasks/ship.rake [hunk 1]
```diff
@@ -180,6 +180,17 @@ namespace :pl do
     end
   end
 
+  desc "ship Windows nuget packages to #{Pkg::Config.nuget_host}"
+  task :ship_nuget => 'pl:fetch' do
+    packages = Dir['pkg/windows/**/*.nupkg']
+    if packages.empty?
+      STDOUT.puts "There aren't any nuget packages in pkg/windows. Maybe something went wrong?"
```
Comment (ferventcoder) [inline tasks/ship.rake]: Are we fine with not bubbling up an error?

**Comment by ferventcoder** [comment]
Probably needs some MOAR documentation. Especially since we are doing it in a very particular way (aka Nexus) and directly posting packages where they would end up if we did the same using choco push to a source.

**Comment by ferventcoder** [comment]
Was this closed on purpose?

**Comment by melissa** [comment]
Yes, meant to be closed.

File: tasks/ship.rake (hunk 1 above)
Comment (melissa) [inline tasks/ship.rake]: We're not always going to have nuget packages to ship, so yes, this is intended behaviour

**Pull request closed without merging**